### PR TITLE
[KOGITO-3966] Update explainability BDD test

### PR DESCRIPTION
**Pull request by r00ta**
Jira: https://issues.redhat.com/browse/KOGITO-3966

The BDD explainability test is currently broken on master due to https://github.com/kiegroup/kogito-examples/pull/477 . 

This PR aims to fix it according to the changes in the dmn model used in the kogito runtime application. 
Many thanks for submitting your 

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-cloud-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains a link to the JIRA issue
- [x] Pull Request contains a description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

**Comment by sutaakar** [comment]
@r00ta Can you please use/create a JIRA for this PR and adjust the PR according to requirements above?

**Comment by r00ta** [comment]
@sutaakar done

File: test/features/install_explainability.feature [hunk 1]
```diff
@@ -64,7 +64,7 @@ Feature: Kogito Explainability service
 
     Then Explainability result for execution "LoanEligibility" in the Trusty service within 3 minutes with saliences:
       | Eligibility  | 
-      | Is Enought?  |
+      | Is Enough?  |
```
Comment (radtriste) [inline test/features/install_explainability.feature:67]: shoudn't you update that one also in the next scenario ?

**Review by radtriste — CHANGES_REQUESTED** [review]

**Comment by r00ta** [comment]
@radtriste can we move forward with this?

**Review by radtriste — APPROVED** [review]

**Pull request merged**